R/FilteredData.R

```diff
@@ -891,6 +945,12 @@ FilteredData <- R6::R6Class( # nolint
     # private attributes ----
     filtered_datasets = list(),
 
+    # turn on / off filter panel
+    filter_turn = TRUE,
```

Comment:
1. I dislike the "turn" term here. You can _turn_ left or right. I propose _state_ / _status_ / _active_ instead. Please change other places as well including method docs.

2. What's the point of this field? I ask from the class simplicity point of view.
I can see that it's used in the api class - there is a check whether it's active and if yes then you can do set/get/etc. Is that needed? The functionality would be anyway disabled from the GUI perspective (vide: disabling inputs). What I am suggesting here is that the big disable/enable button would _just_ propagate this action to all relevant inputs + save/restore action and that's it. An object would not store its state as this opens a door for unnecessary complexity I believe. I am happy to discuss it further if needed.

Reply:
By default `shinyWidgets::switchInput` use the logical(1) for value so this should not be surprising I stay with this type.

`filter_turn` variable is needed only for filter_panel_api, so we could provide a proper warning if sb want to update the state when filter panel is turn off.

I like the Turn Off/On naming, although I see you agree with Nik in this area so please update to the one you are preferring.